### PR TITLE
Simple phrasing changes

### PR DESCRIPTION
**Pull request by nstanard**
Before submitting a pull request, please make sure you have at least performed the following:

 - [] read and followed the [CONTRIBUTING.md](https://github.com/ng-bootstrap/ng-bootstrap/blob/master/CONTRIBUTING.md) guide.
 - [] built and tested the changes locally.
 - [] added/updated any applicable tests.
 - [] added/updated any applicable API documentation.
 - [] added/updated any applicable demos.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ Once installed you need to import our main module:
 ```js
 import {NgbModule} from '@ng-bootstrap/ng-bootstrap';
 ```
-The only remaining part is to list the imported module in your application module. The exact method will be slightly
```
Comment (pkozlowski-opensource) [inline README.md]: This changes the meaning of this sentence, really. What we are talking about here is the main module of an application.

**Comment by pkozlowski-opensource** [comment]
LGTM apart from one comment I've left. Could you please fix it, squash commits and amend the commit message to follow our conventions?  (https://github.com/ng-bootstrap/ng-bootstrap/blob/master/CONTRIBUTING.md#you-want-to-contribute-some-code)

Thnx!

**Comment by nstanard** [comment]
ummm.... Is it a problem I merged from master before squashing @pkozlowski-opensource?

**Comment by pkozlowski-opensource** [comment]
@nstanard yeh, your PR is messed up a bit atm :-) Try to clean it up and / or open a new one if git gives you headaches.

**Comment by nstanard** [comment]
Nevar! This is a good chance for me to learn more about git. I will fix this in a bit.

**Comment by nstanard** [comment]
BOOM! @pkozlowski-opensource 

How's that message format look?

**Comment by pkozlowski-opensource** [comment]
> How's that message format look?

I've changed it slightly and merged the PR, thnx!

> Nevar! This is a good chance for me to learn more about git. I will fix this in a bit.

Ha! That's the spirit! Keep it up!

**Pull request closed without merging**